core/bin/zksync_tee_prover/src/config.rs

```diff
@@ -0,0 +1,44 @@
+use std::path::PathBuf;
+
+use secp256k1::SecretKey;
+use url::Url;
+use zksync_env_config::FromEnv;
+use zksync_types::tee_types::TeeType;
+
+/// Configuration of the TEE prover, which verifies
+/// [`zksync_prover_interface::inputs::TeeVerifierInput`]s using
+/// [`zksync_tee_verifier::Verifiable::verify()`]. It signs the correctly verified inputs with
+/// [`TeeProverConfig::signing_key`] and submits the proofs to the TEE prover API at
+/// [`TeeProverConfig::api_url`].
```

Comment:
I over-documented because I thought putting `Configuration for the TEE prover` above `struct TeeProverConfig` doesn't make much sense, as the name of the `TeeProverConfig` is self-explanatory.

Also, I'm not a native speaker, but I assumed `which` in this case refers to the TEE prover, not to the configuration of the TEE prover. I asked ChatGPT  _![Screenshot from 2024-07-01 20-16-09](https://github.com/matter-labs/zksync-era/assets/3036218/65cec4a2-2060-47c7-b93c-601e8cb26b9f)_ referring to in this sentence?`

Anyway, I'will fix it.

Reply:
ChatGPT seems to be confused when you ask it to rewrite the text, and now suddenly configuration _is_ responsible for all these things 😅

![image](https://github.com/matter-labs/zksync-era/assets/12111581/d7d99929-c462-4261-be22-418fba7821af)